bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml

```diff
@@ -23,7 +23,21 @@
 
 		<parameter name="gems" type="text" required="false" groupName="gems">
 			<label>Ruby Gems</label>
-			<description>A comma separated list of Ruby Gems to install.</description>
+			<description><![CDATA[A comma separated list of Ruby Gems to install. Versions may be constrained by separating with an
+			    <tt>=</tt> and then the standard RubyGems version constraint, such as <tt>openhab-scripting=~>4.0</tt>.
+				]]></description>
+		</parameter>
+
+		<parameter name="check_update" type="boolean" required="true" groupName="gems">
+			<label>Check for Gem Updates</label>
+			<description>Check RubyGems for updates to the above gems when OpenHAB starts or JRuby settings are changed.
+				Otherwise it will try to fulfill the requirements with locally installed gems, and you can manage them yourself with
+				an external Ruby by setting the same GEM_HOME.</description>
+			<options>
+				<option value="true">Check For Updates</option>
+				<option value="false">Do Not Check For Updates</option>
+			</options>
+			<default>true</default>
```

Comment:
The i18n translations file need to be updated too

I usually use this command while inside the /bundles/org.openhab.automation.jrubyscripting directory

```
mvn org.openhab.core.tools:i18n-maven-plugin:3.3.0:generate-default-translations -pl :org.openhab.automation.jrubyscripting```

Reply:
Thanks. I was wondering what it took to get my OpenHAB to update these... I figured it was just a cache.